Makefile

```diff
@@ -0,0 +1,24 @@
+PREFIX := /etc/openvpn
+DESTDIR := $(PREFIX)/scripts
+
+SRC = update-systemd-resolved
+DEST = $(DESTDIR)/$(SRC)
+
+.PHONY: all install info
+
+all: install info
```

Comment:
Out of interest, is there any reason you didn't include a `test` stage in the Makefile?